src/AddTodoForm.js

```diff
@@ -12,21 +11,20 @@ const AddTodoForm = ({ onAddTodo }) => {
 
   const handleAddTodo = (event) => {
     event.preventDefault();
+
     if (!todoTitle) {
```

Comment:
Good job implementing some form validation here! This is great for user experience! 

Since you're not returning anything, this is a good time to implement a "oneliner"  for this code block.